recipes/sbi/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "sbi" %}
+{% set version = "0.23.2" %}
+{% set python_min = "3.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/sbi-{{ version }}.tar.gz
+  sha256: 96b6dea2acb410d871bb2ea5b6c0b487aa6d270ac35900b4859981af4a2eab7c
+
+build:
+  noarch: python
```

Comment:
Windows builds will fail here as expected, given that PyTorch doesn't support Windows. The rest is `noarch` though so can keep it as `noarch: python`.